test_scripts/test_generate_questions.py

```
@@ -0,0 +1,37 @@
+import sys
```

Ruff

`sys` imported but unused

Remove unused import: `sys`

(F401)